easybuild/easyblocks/p/petsc.py

```diff
@@ -312,10 +317,18 @@ def sanity_check_step(self):
         else:
             libext = "a"
 
-        custom_paths = {
-                        'files': [os.path.join(prefix2, "lib", "libpetsc.%s" % libext)],
-                        'dirs': [os.path.join(prefix1, "bin"), os.path.join(prefix2, "conf"),
+        if LooseVersion(self.version) < LooseVersion("3.6"):
+            custom_paths = {
+                            'files': [os.path.join(prefix2, "lib", "libpetsc.%s" % libext)],
+                            'dirs': [os.path.join(prefix1, "bin"), os.path.join(prefix2, "conf"),
```

Comment:
Yes, it seems that ```conf``` dir is not longer there. From my point of view this folder can be removed from the sanity check. I added these lines of code for legacy reasons.